setup_dev_env.sh

```diff
@@ -30,6 +30,8 @@ Usage: $0
     [--git-name <name>]             use the provided name for git commits
     [--git-email <email>]           use the provided email for git commits
     [--git-editor <editor command>] use the provided editor for git
+    [--program <path>]              run the provided program instead of bash
+                                    for example: /stratum/tools/run_unit_tests.sh
```

Comment:
I see why you want to add this command, but the `setup_dev_env.sh` script is not for CI.

You can just use the image normally, i.e. `docker run` and set a different [entrypoint](https://docs.docker.com/engine/reference/commandline/run/).
The details depend on your CI system, but most should support something like this: https://github.com/stratum/stratum/blob/68c8bf999baf256af23f530e36cec060b9919d49/.circleci/config.yml#L79-L80

Considering that the Stratum CI will not use this flag, I'm against adding this flag.